<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleQtVTK</class>
 <widget class="QWidget" name="SimpleQtVTK">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1424</width>
    <height>1036</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1" colspan="16">
    <widget class="QVTKWidget" name="vtkWidget" native="true"/>
   </item>
   <item row="2" column="10">
    <widget class="QCheckBox" name="displaceMeshBox">
     <property name="text">
      <string>Displace mesh</string>
     </property>
    </widget>
   </item>
   <item row="4" column="10">
    <widget class="QCheckBox" name="showAxesBox">
     <property name="text">
      <string>Show axes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Tri alpha</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="showScaleBox">
     <property name="text">
      <string>Show scale</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="fieldsCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Opacity</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="fileCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="resetScaleButton">
     <property name="text">
      <string>Reset scale</string>
     </property>
    </widget>
   </item>
   <item row="4" column="9">
    <widget class="QCheckBox" name="autoTriAlphaBox">
     <property name="text">
      <string>Auto tri alpha</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <widget class="QCheckBox" name="printColorBox">
     <property name="text">
      <string>Print color scheme</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Field</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="resetViewButton">
     <property name="text">
      <string>Reset view</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QCheckBox" name="autoScaleBox">
     <property name="text">
      <string>Auto scale</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QCheckBox" name="showMeshBox">
     <property name="text">
      <string>Show mesh</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QDoubleSpinBox" name="triAlphaSpin"/>
   </item>
   <item row="4" column="4">
    <widget class="QCheckBox" name="showPointsBox">
     <property name="text">
      <string>Show points</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QDoubleSpinBox" name="opacitySpin"/>
   </item>
   <item row="4" column="11">
    <widget class="QPushButton" name="screenShotButton">
     <property name="text">
      <string>Screen capture</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvtkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
